### PR TITLE
Add a note about changes in importing core and model classes

### PR DESCRIPTION
**Pull request by nhanitvn**

### CODE REVIEW

File: USAGE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # USAGE NOTES
```
Comment (Shiti) [inline USAGE_NOTES.md:1]: I think we should rename the file to `migrating.md`

File: USAGE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-## July 16th 2020
+## July 16th 2021
```
Comment (Shiti) [inline USAGE_NOTES.md:2]: Please clarify here what the migration is from and to what. For example, `upgrading from version 2020.8 to 0.0.3` or something similar.

File: USAGE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 Due to changes to the package structure, please update your code to:
 - Import `Graph` class from `h1st.h1flow.h1flow` instead `h1st.core.graph` .
 - Import `Node` class from `h1st.h1flow.h1step` instead of `h1st.core.node`.
```
Comment (Shiti) [inline USAGE_NOTES.md:5]: I think the imports for `Action`, `Decision`, etc may also need to be updated

**Review by Shiti — CHANGES_REQUESTED** [review]

File: USAGE_NOTES.md (hunk 1 above)
Comment (nhanitvn) [reply USAGE_NOTES.md]: I renamed to MIGRATION_NOTES.md. Is it better now?

File: USAGE_NOTES.md (hunk 2 above)
Comment (nhanitvn) [reply USAGE_NOTES.md]: Done

File: USAGE_NOTES.md (hunk 3 above)
Comment (nhanitvn) [reply USAGE_NOTES.md]: Done

**Review by Shiti — APPROVED** [review]

**Pull request merged**